test/unit/chat/bookmarks.test.ts

```diff
@@ -1,11 +1,18 @@
 import { silenceDebugMessages } from '../../helpers/setup'
 import { findBookmarkDocument, toggleBookmark, renderBookmarksButton } from '../../../src/chat/bookmarks'
-import { NamedNode } from 'rdflib'
+import { NamedNode, Namespace } from 'rdflib'
+import store from '../../../src/store'
+import { clearStore } from '../helpers/clearStore'
+import { ns } from '../../../src/'
+
+const BOOK = Namespace('http://www.w3.org/2002/01/bookmark#')
 
 silenceDebugMessages()
 jest.mock('solid-auth-client')
 
 describe('findBookmarkDocument', () => {
+  afterEach(clearStore)
+
   it('exists', () => {
```

Comment:
I think we were changing this to something like ('it is exposed on the API'..  you can check out tabs.test.ts.